NETCORE/src/Microsoft.ApplicationInsights.AspNetCore/DiagnosticListeners/Implementation/HostingDiagnosticListener.cs

```diff
@@ -229,7 +229,9 @@ public void OnHttpRequestInStart(HttpContext httpContext)
                 {
                     // Scenario #3. W3C-TraceParent
                     // We need to ignore the Activity created by Hosting, as it did not take W3CTraceParent into consideration.
+#pragma warning disable CA2000 // Dispose objects before losing scope
                     newActivity = new Activity(ActivityCreatedByHostingDiagnosticListener);
+#pragma warning restore CA2000 // Dispose objects before losing scope
```

Comment:
need to include a comment explaining why we're not disposing this Activity

Reply:
Just added. Let me know what you think. Thanks for reviewing!

Reply:
the comment seems incorrect. We dispose it in the OnEnd callback. We get access to this activity, by using Activity.Current.

Reply:
in which file are we disposing the Activity.Current?

Reply:
I actually meant we are stopping the activity, not dispose. But dispose is doing nothing but stop!

Reply:
https://github.com/dotnet/runtime/blob/master/src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.cs#L871-L885

Reply:
@cijothomas @TimothyMothra , just updated the comments!